app/src/main/res/values/dimens.xml

```diff
@@ -62,10 +62,20 @@
   <dimen name="recently_played_margin_min">8dp</dimen>
   <dimen name="recently_played_tablet_land_outer_margin">96dp</dimen>
   <dimen name="recently_played_tablet_port_outer_margin">120dp</dimen>
-  <dimen name="default_reading_text_size">18sp</dimen>
 
+  <!-- Scalable Pixels for Text-->
+  <dimen name="default_reading_text_size">18sp</dimen>
   <dimen name="font_scale_content_text_size">20dp</dimen>
   <dimen name="profile_view_already_added_margin">20dp</dimen>
   <dimen name="profile_view_not_added_margin">20dp</dimen>
   <dimen name="walkthrough_topic_summary_container_margin">20dp</dimen>
+  <dimen name="default_button_text_size">14sp</dimen>
+  <dimen name="heading1_text_size">24sp</dimen>
+  <dimen name="heading2_text_size">20sp</dimen>
+  <dimen name="heading3_text_size">18sp</dimen>
+  <dimen name="heading4_text_size">18sp</dimen>
+  <dimen name="heading5_text_size">14sp</dimen>
+  <dimen name="subtitle1_text_size">16sp</dimen>
+  <dimen name="subtitle2_text_size">14sp</dimen>
```

Comment:
This is not needed. Please revert the changes in this file.

Reply:
Reverted